test/teslamate_web/controllers/car_controller_test.exs

```diff
@@ -316,6 +317,7 @@ defmodule TeslaMateWeb.CarControllerTest do
              charging_state: "Charging",
              charge_energy_added: "4.32",
              ideal_battery_range: 200,
+             charge_rate: 123,
```

Comment:
I wonder if 123 is a realistic charge rate

Reply:
See previous comments

Reply:
as before